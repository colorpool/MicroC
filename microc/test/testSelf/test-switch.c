void main(int n) {
  
    switch( n ){
        case 1 : {print 'd' : 1; break;}
        case 2 :{ print 'd' : 2; break;}
        default : print 'd' : 3;
    }

    print 'd' : n;

}