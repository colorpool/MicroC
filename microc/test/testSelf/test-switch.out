24 19 1 5 25 13 11 0 1 6 17 19 0 1 22 15 -1 16 45 13 11 0 2 6 17 33 0 2 22 15 -1 16 45 13 11 13 11 6 17 45 0 3 22 15 -1 13 11 22 21 1